about.html

```diff
@@ -0,0 +1,184 @@
+                </div>
+                <li class="nav-item">
+                  <a
+                    class="nav-link active"
+                    aria-current="page"
+                    href="index.html"
+                    >About</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Main Event</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Sponsor Event</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Coming Event</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Get Ticket</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+      </menu>
+    </header>
+    <section id="top-banner">
+      <div
+        class="container about-headline d-flex flex-column justify-content-center align-items-center"
+      >
+        <h5 class="event-head-1 secondary-color mb-0 d-none d-md-block">
+          "Welcome To The"
+        </h5>
+        <h1 class="event-headline-main text-uppercase text-center">
+          Night of Recovery Concert
+        </h1>
+        <div class="about-title-line mb-4"></div>
+        <div class="event-summary">
+          <p class="text-center">
+            The Night of Recovery Concert is a celebration of strength,
+            resilience, and the power of music in healing. This captivating
+            event brings together renowned musicians, inspiring speakers, and a
+            community of individuals who believe in the transformative power of
+            recovery. Join us for an unforgettable night as we unite to support
+            those on their journey towards healing and overcoming addiction.
+          </p>
+        </div>
+        <p class="text-center">
+          Please contact us via Email for any further questions about The Night
+          of Recovery Concert!<br />
+          <a href="mailto:" class="primary fw-bold mail-event"
+            >moreinfo@thenightofrecovery.org</a
+          >
+          >
+        </p>
+      </div>
+    </section>
+    <section id="about-logo">
+      <div
+        class="container d-flex flex-column justify-content-center align-items-center about-logo py-4 py-md-0"
+      >
+        <h2 class="text-center primary">
+          Ticket Information About The Concert
+        </h2>
+        <div class="about-title-line mb-4"></div>
+        <p class="text-center primary">
+          Tickets for the Night of Recovery Concert are available for purchase
+          online through our website. Stay tuned for updates on ticket prices,
+          early bird discounts, and special packages.
+        </p>
+        <div class="logo-wrap border p-5">
+          <img src="images/logo.png" alt="concert logo" class="w-100" />
+        </div>
```

Comment:
- Hey, great job adding the About page as pointed out by the previous reviewer, shower, let's consider reducing the width of the logo here in the About page for the mobile and desktop versions to improve your design as compared to the main design.

*Your design*
![image](https://github.com/sulaimonaa/music-concert/assets/59397159/6ac74875-ffd6-4971-bf50-128e6eaa6202)


*Main design*
![image](https://github.com/sulaimonaa/music-concert/assets/59397159/b04a5671-a3b9-48d5-9ec1-a6337882c146)